.expeditor/config.yml

```diff
@@ -13,18 +13,20 @@ slack:
 rubygems:
   - train-habitat
 
-promote:
-  actions:
-    - built_in:rollover_changelog
-    - built_in:publish_rubygems
+subscriptions:
```

Comment:
There should only be one `subscriptions` block — this needs to be combined with the one below.